examples/config-sync.json

```diff
@@ -30,6 +30,11 @@
 							"semver":true
 						}
 					},
+					{
+						"prefix":"/repo1/repo",
+						"destination": "/repo",
+						"stripPrefix": true
```

Comment:
Does stripPrefix mean you only have "/repo" and images land there instead of "/repo/repo1/repo"

Reply:
Yes, exactly.